chain/sub/incoming.go

```diff
@@ -146,8 +144,7 @@ func (bv *BlockValidator) Validate(ctx context.Context, pid peer.ID, msg *pubsub
 	// track validation time
 	begin := time.Now()
 	defer func() {
-		end := time.Now()
-		log.Infof("block validation time: %s", end.Sub(begin))
+		log.Debugf("block validation time: %s", time.Since(begin))
```

Comment:
@placer14 thoughts on making this a `Debugw`?

Reply:
I'm okay with it. I've added it to opencensus as well, so we could remove it completely. Only concern with removing it is that polling via opencensus doesn't allow all individual measurements to be captured, only the measurement available at the time of the polling. Debugw is a reasonable balance. 🤝

Reply:
Might be better to `Debugw("block validation", "took", time.Since(begin))` to match the patterns elsewhere.